docs/user_docs/maintaince/scale/vertical-scale.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Vertical Scale
+description: How to vertically scale a cluster
+keywords: [vertical scale, vertical scaling]
+sidebar_position: 1
+sidebar_label: Vertical Scale
+---
+
+# Vertical Scale
+
+You can change the resource requirements and limits (CPU and storage) by performing the vertical scale. For example, you can change the resource class from 1C2G to 2C4G.
+
+This tutorial takes MySQL as an example to illustrate how to vertically scale a cluster.
+
+:::note
+
+- During the vertical scaling process, all pods restart and the pod role may change after the restarting.
```

Comment:
KubeBlocks now supports "In-place pod resource resizing" which doesn't restart vertical-scaling Pods for K8s version >= 1.27 and InPlacePodVerticalScaling feature gate enabled, @free6om, right?